From f2b283f4536b5a4b9c0aae23cd97d65af9273389 Mon Sep 17 00:00:00 2001
From: David Gibson <dgibson@redhat.com>
Date: Mon, 21 Jul 2014 04:45:17 +0200
Subject: [PATCH 208/212] Include OHCI device for ppc64

Message-id: <1405917919-7547-2-git-send-email-dgibson@redhat.com>
Patchwork-id: 59980
O-Subject: [PATCH 1/3] Include OHCI device for ppc64
Bugzilla:
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

With the current RHEL7 qemu configuration, UHCI, EHCI and XHCI devices are
included on all platforms, but not OHCI.  But on real hardware pp64
machines (and in fact most non-x86 platforms), OHCI is the standard USB1
controller, and UHCI is almost unheard of.

Arguably we should replace UHCI with OHCI on all non-x86 platforms.  But
for now just enable OHCI for ppc64, where we definitely need it.

Signed-off-by: David Gibson <dgibson@redhat.com>
---
 default-configs/ppc64-softmmu.mak | 1 +
 1 file changed, 1 insertion(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 default-configs/ppc64-softmmu.mak |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/default-configs/ppc64-softmmu.mak b/default-configs/ppc64-softmmu.mak
index a9798a8..c122b25 100644
--- a/default-configs/ppc64-softmmu.mak
+++ b/default-configs/ppc64-softmmu.mak
@@ -58,3 +58,4 @@ CONFIG_I8257=y
 CONFIG_MC146818RTC=y
 CONFIG_ISA_TESTDEV=y
 CONFIG_LIBDECNUMBER=y
+CONFIG_USB_OHCI=y
-- 
1.7.1

